spec/requests/pingometer_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.describe 'Pingometer Webhook', type: :request do
+  it 'is persisted appropriately in the database' do
+    monitor_id = 'b74014410cc1236a3d0h7400'
+
+    # Expect a pre-existing monitor
+    web_service = WebService.create pingometer_id: monitor_id,
```

Comment:
Sorry, the TL;DR there is that we should expect a monitor with the same *hostname*, but not necessarily with the same *ID*.